pkg/config/config.go

```diff
@@ -35,6 +38,13 @@ func NewConfig(configPath string) error {
 		return err
 	}
 
+	// clear out flavours & podTemplate key so koanf does not try to unmarshal them later
```

Comment:
if you have an koan-issue or a link to a documentation by hand, would be nice to have it here - or you just describe a bit more better why we do not want it here

Reply:
done with da438f7